programs/ziti-edge-tunnel/windows-scripts.c

```diff
@@ -302,15 +301,38 @@ void remove_nrpt_rules(uv_loop_t *nrpt_loop, model_map *hostnames) {
     }
 }
 
-void remove_all_nrpt_rules() {
+char* get_nrpt_comment(char* zet_id, bool exact) {
```

Comment:
`zet_id` can be const. also I think you could drop the `exact` parameter and use `zet_id != NULL` instead.